test/accounts_test.go

```diff
@@ -177,28 +302,57 @@ func TestUpdateEntity(t *testing.T) {
 	}
 }
 
-func createEntity() string {
-	r := accounts.OnboardEntityRequest{
-		Reference:      GenerateRandomReference(),
-		ContactDetails: &accounts.ContactDetails{Phone: &accounts.Phone{Number: "2345678910"}},
-		Profile: &accounts.Profile{
-			Urls: []string{"https://www.superheroexample.com"},
-			Mccs: []string{"0742"},
+func TestQueryPaymentInstruments(t *testing.T) {
+	var (
+		entityId = createEntityCompany()
+		query    = accounts.PaymentInstrumentsQuery{
+			Status: accounts.Unverified,
+		}
+		file = accounts.File{
+			File:    "./checkout.pdf",
+			Purpose: common.BankVerification,
+		}
+
+		requestFileId = submitFile(file)
+
+		_ = paymentInstrumentRequest(entityId, requestFileId)
```

Comment:
I don't understand this line, is not being used anywhere, right?

Reply:
Removed